### DISCUSSION TITLE
How do I create a table in a schema that is not 'public'?

### DISCUSSION THREAD

**Discussion by batazor** [prisma/prisma#8803] [Q&A] [answered]
I want the migration schema to be stored in `public` 
while the table itself was created in another schema, what is it:

```sql
CREATE TABLE billing.account(
    "id" UUID NOT NULL,
    "user_id" UUID NOT NULL,
    "tariff_id" UUID NOT NULL
);
```

I can of course manually overwrite the tables, but is there a better way?

```sql
-- Move this table
ALTER TABLE account SET SCHEMA billing;
```

**Comment by janpio** [comment, marked as answer]
Prisma right now only supports working with tables stored in one common schema on PostgreSQL. As soon as you have them in multiple schemas, Prisma will have trouble reaching those tables.

**Reply by batazor** [reply]
I understand, thank you for the answer.

**Reply by janpio** [reply]
Here is a feature request for multi schema support btw: https://github.com/prisma/prisma/issues/1122

**Reply by ryands17** [reply]
@batazor It would be great if you could add a 👍 to the above request 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
